src/taskManager.js

```diff
@@ -0,0 +1,35 @@
+// eslint-disable-next-line import/no-mutable-exports
+let tasks = [];
+
+function getTasks() {
+  const savedTasks = localStorage.getItem('tasks');
+  if (savedTasks) {
+    tasks = JSON.parse(savedTasks);
+  }
+}
+
+function saveTasks() {
+  localStorage.setItem('tasks', JSON.stringify(tasks));
+}
+
+function addTask(description) {
+  const task = {
+    description,
+    completed: false,
+    index: tasks.length + 1,
+  };
+
+  tasks.push(task);
+}
+
+function removeTask(index) {
+  tasks.splice(index, 1);
+}
```

Comment:
> Deleting a task should update all remaining items' indexes, so they represent the current list order and are unique(i.e. if you're deleting the first task index 1 from the list, the index of the next task(2) should set to 1)..
- [ ] According to the project requirement, deleting a task should rearrange the indexes of the previous tasks. Currently, you haven't implanted this in your code. I'll suggest you write out a function that does this and call the function after deleting each task. :+1: :smile: